fedimint/src/lib.rs

```diff
@@ -322,12 +395,6 @@ impl FedimintServer {
                 .send(msg.target, EpochMessage::Continue(msg.message))
                 .await;
         }
-
-        while outcomes.is_empty() {
-            let msg = self.connections.receive().await;
-            outcomes = self.handle_message(msg).await;
-        }
-        outcomes
```

Comment:
This removes handling outcomes that are returned on proposing. For real federations this isn't expected (but should still be handled just in case), for fake 1-of-1 ones this could be problematic.

Reply:
This is just a refactor that separates out the send because it's used twice now.  In both cases the outcomes will be processed, just the rejoin needs to process all the buffered messages first.

Alternatively, I could have passed the buffered messages into this function to be processed first, but felt that was more confusing.

Reply:
I don't see where a potential outcome generated here would be processed

https://github.com/fedimint/fedimint/blob/d666f79e5490268a09f49ac5bd127bdca3409086/fedimint/src/lib.rs#L388-L391